cloudferrylib/os/compute/nova_compute.py

```diff
@@ -383,6 +383,16 @@ def convert_instance(instance, cfg, cloud):
         else:
             server_group = None
 
+        config_drive = instance.config_drive
```

Comment:
because it contains a good implementation of converters from string to any other types.
And i already used the oslo_config library to solve similar problem.